server/services/licenceService.ts

```diff
@@ -56,8 +58,10 @@ export default class LicenceService {
 
     const offenderManager = deliusRecord.offenderManagers.find(om => om.active)
 
+    const licenceType = this.getLicenceType(nomisRecord)
+
```

Comment:
This is the part we may need to migrate to, or duplicate in, the caseload service for showing on the case list. Ok here for now.